src/corehost/cli/runtime_config.cpp

```diff
@@ -29,6 +29,16 @@ void runtime_config_t::parse(const pal::string_t& path, const pal::string_t& dev
         m_prerelease_roll_fwd = defaults->m_prerelease_roll_fwd;
         m_roll_fwd_on_no_candidate_fx = defaults->m_roll_fwd_on_no_candidate_fx;
     }
+    else
+    {
+        // Since there is no previous config, this is the app's config, so default m_roll_fwd_on_no_candidate_fx from the env variable.
```

Comment:
This fixes a bug (along with code removed later on): we only want the first config (the app's config) to be defaulted from the env variable. Then the first framework will be defaulted from the app's config, but the framework can override in its config. The next, etc. framework will default from the previous framework but can still override in its config as well.

Most likely framework's won't override this setting, and is not likely a user will change it since it may require admin rights at least if it's in the shared Program Files location.